product_docs/docs/pgd/5/ddl/ddl-overview.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: DDL overview
+navTitle: Overview
+---
+
+DDL stands for data definition language, the subset of the SQL
+language that creates, alters, and drops database objects.
+
+## Replicated DDL
+
+For operational convenience and correctness, PGD replicates most DDL
+actions, with these exceptions:
+
+-   Temporary or unlogged relations
+-   Certain DDL statements (mostly long running)
+-   Locking commands (`LOCK`)
+-   Table maintenance commands (`VACUUM`, `ANALYZE`, `CLUSTER`, `REINDEX`)
+-   Actions of autovacuum
+-   Operational commands (`CHECKPOINT`, `ALTER SYSTEM`)
+-   Actions related to databases or tablespaces
+
+Automatic DDL replication makes 
+certain DDL changes easier without your having to manually distribute
```

Comment:
I just learned about this language construct "your having to". Sounds weird to my ears, but seems correct. Always learning... :-)

Reply:
That said, it would be an easier read with...

```suggestion
certain DDL changes easier without having to manually distribute
```